posthog/migrations/0156_insight_short_id.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.1.8 on 2021-05-26 23:18
+
+from django.db import migrations, models
+
+import posthog.models.dashboard_item
+
+
+def create_short_ids(apps, schema_editor):
+    DashboardItem = apps.get_model("posthog", "DashboardItem")
+    for obj in DashboardItem.objects.all():
+        obj.short_id = posthog.models.dashboard_item.generate_short_id()
```

Comment:
NIT: import function above

Reply:
I believe the import function is still needed to be able to generate the `short_id` (this was the default behavior from the migration generator)